fiftyone/core/expressions.py

```diff
@@ -545,10 +545,10 @@ def re_match(self, regex, options=None):
         Args:
             regex: the regular expression to apply. Must be a Perl Compatible
                 Regular Expression (PCRE). See
-                `this page <https://docs.mongodb.com/manual/reference/operator/aggregation/regexMatch/#regexmatch-regex>`_
+                `this page <https://docs.mongodb.com/manual/reference/operator/aggregation/regexMatch/#regexmatch-regex>`__
```

Comment:
why is double underscore necessary?

Reply:
https://stackoverflow.com/questions/27420317/restructured-text-rst-http-links-underscore-vs-use

These links were previously also creating references named "this page", which led to warnings because they were duplicated (partly because there were two occurrences here, and partly because this docstring is inherit by a child class).

https://github.com/voxel51/fiftyone/runs/1058667060?check_suite_focus=true#step:6:176

```
2020-09-01T22:00:28.1320372Z /home/runner/work/fiftyone/fiftyone/fiftyone/core/expressions.py:docstring of fiftyone.core.expressions.ViewField.trunc:19: WARNING: Duplicate explicit target name: "this page".
2020-09-01T22:00:28.1321519Z /home/runner/work/fiftyone/fiftyone/fiftyone/core/expressions.py:docstring of fiftyone.core.expressions.ViewField.trunc:19: WARNING: Duplicate explicit target name: "this page".
2020-09-01T22:00:28.1322703Z /home/runner/work/fiftyone/fiftyone/fiftyone/core/expressions.py:docstring of fiftyone.core.expressions.ViewField.trunc:19: WARNING: Duplicate explicit target name: "this page".
```

Reply:
Ah I see. Didn't know about anonymous links. Thanks!